tests/js/spec/views/settings/components/dataScrubbing/dataScrubbing.spec.tsx

```diff
@@ -0,0 +1,250 @@
+import React from 'react';
+
+import {mountWithTheme} from 'sentry-test/enzyme';
+
+import DataScrubbing from 'app/views/settings/components/dataScrubbing';
+import {ProjectSlug} from 'app/views/settings/components/dataScrubbing/types';
+import {addSuccessMessage} from 'app/actionCreators/indicator';
+
+jest.mock('app/actionCreators/indicator');
+
+const relayPiiConfig = {
+  rules: {
+    '0': {type: 'password', redaction: {method: 'replace', text: 'Scrubbed'}},
+    '1': {type: 'creditcard', redaction: {method: 'mask'}},
+  },
+  applications: {password: ['0'], $message: ['1']},
+};
+
+const stringRelayPiiConfig = JSON.stringify(relayPiiConfig);
+
+const rules = [
+  {
+    id: 0,
+    method: 'replace',
+    placeholder: 'Scrubbed',
+    source: 'password',
+    type: 'password',
+  },
```

Comment:
Can we add this to `tests/js/sentry-test/fixtures` so that we can call `TestStubs.DataScrubbingRule()`?

Reply:
good idea!!